pkg/sql/planner.go

```diff
@@ -281,8 +281,8 @@ func newInternalPlanner(
 		ctx, sd, dataMutator, tables, txn, ts, ts, execCfg, &plannerMon,
 	)
 	p.extendedEvalCtx.Planner = p
-	p.extendedEvalCtx.PrivilegedAccessor = p
 	p.extendedEvalCtx.SessionAccessor = p
+	p.extendedEvalCtx.PrivilegedAccessor = p
```

Comment:
[nit]: unnecessary change

Reply:
i'm just tryna keep the order the same hahahah